src/app/helptext/storage/volumes/volume-list.ts

```diff
@@ -55,7 +55,11 @@ pool_lock_warning_paratext_b: T("</i>.\
  Data on the pool will not be accessible until the pool is unlocked."),
 
 permissions_edit_msg: T('Root dataset permissions cannot be edited.'),
-acl_edit_msg: T('Root dataset ACL cannot be edited.')
-
+acl_edit_msg: T('Root dataset ACL cannot be edited.'),
 
+expand_pool_dialog: {
+    title: T("Enter passphrase to expand pool "),
```

Comment:
```suggestion
    title: T("Enter passphrase to expand pool"),
```

Reply:
Thanks @Mrt134,  the white space is in purpose, the pool name will append to it.